x-pack/plugins/infra/public/alerting/inventory/components/expression.tsx

```diff
@@ -70,6 +70,7 @@ import { useKibanaContextForPlugin } from '../../../hooks/use_kibana';
 
 import { ExpressionChart } from './expression_chart';
 const FILTER_TYPING_DEBOUNCE_MS = 500;
+export const QUERY_INVALID = Symbol('QUERY_INVALID');
```

Comment:
Nit: For me a name like `INVALID_QUERY_MARKER` would be more clear.

Reply:
Would it make sense to use `unique symbol` type and export that type to the other places where this is the value being passed in? So it's not "any symbol"?